Dockerfile

```diff
@@ -1,10 +1,4 @@
-FROM amd64/ubuntu:bionic@sha256:8da4e9509bfe5e09df6502e7a8e93c63e4d0d9dbaa9f92d7d767f96d6c20a78a
-RUN echo "disabled"
-
-FROM arm64v8/ubuntu:bionic@sha256:6a4619c02fbaf80504f316f42bd4b732831d9590e9c1c0df2b6f294ffbee86c9
-RUN echo "disabled"
-
-FROM ubuntu:bionic
+FROM ubuntu:focal
```

Comment:
I believe that if we're a bit more specific here then we'll give a chance for dependabot to bump it for us, right? Otherwise we'll get a different base image every time the build is launched and the `focal` image change (since that's just a pointer the the latest `focal` image (20.04) and not to a specific semver

Reply:
yes, we may have this opportunity, but I thought that every time the image is bumped, a new release has to be created out of it, and we should bump the httpbin image in KIC.  For an image used only for testing purposes, I wonder if it is worth this kind of effort. WDYT?

Reply:
Ok, so we don't **have** to bump the image in KIC every time the base image gets bumped here.

It would be a very nice thing to have but not strictly necessary. OTOH though, with dependabot set in KIC looking through docker images to get a bump I believe it would issue a PR automatically to update httpbin in KIC so we'd get it "for free".

I believe it will be better than having a vague version pointer (this is basically what `focal` is) and wondering each time the image gets built, what's the base image version.

Reply:
I get your point, but tbh I think that having such a workflow also in KIC to bump httpbin and go-echo images can be a little overkill, especially considering the purpose of these images. I propose another solution: we can use a specific tagged version of Focal (the latest is https://hub.docker.com/layers/ubuntu/library/ubuntu/focal-20220531/images/sha256-b2339eee806d44d6a8adc0a790f824fb71f03366dd754d400316ae5a7e3ece3e?context=explore) so that we exactly know the version that we are running. wdyt?